Source/Python/WarpXWrappers.cpp

```diff
@@ -606,6 +606,18 @@ namespace
         particle_buffers.clearParticles();
     }
 
+    void warpx_clearChargeDensity (int lev) {
```

Comment:
Note that this isn't really needed, since one can set the `rho_fp` array using the wrapper object, `RhoFPWrapper`. It should work to do `self.rho_wrapper[Ellipsis] = 0.`.

Reply:
Thanks! I didn't think of that but it is definitely better to reuse functionality already present.